src/styles/blocks/card.scss

```diff
@@ -0,0 +1,143 @@
+/* #region card */
+.card {
+  display: flex;
+  flex-direction: column;
+
+  margin: 0;
+  width: $card-width;
+
+  box-sizing: border-box;
+  border: $border-width solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+
+    margin-left: 19px;
+    margin-top: 32px;
+    margin-bottom: 24px;
+  }
+
+  &__product-item {
+    width: 166px;
+    height: 36px;
+    line-height: 18px;
+
+    margin-top: 16px;
+    margin-left: 16px;
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+
+    color: rgba(6, 11, 53, 1);
+  }
+
+  &__product-id {
+    display: flex;
+    justify-content: center;
+
+    width: 94px;
+    height: 14px;
+    margin: 0 14px 16px;
+
+    font-weight: 400;
+    font-size: 10px;
+    font-style: normal;
+    color: rgba(97, 96, 112, 1);
+  }
+
+  &__product-review {
+    display: flex;
+
+    height: 16px;
+
+    margin-top: 0;
+    margin-right: 16px;
+    margin-left: 16px;
+    margin-bottom: 24px;
+  }
+
+  &__review-feedback {
+    display: flex;
+    align-items: center;
+
+    margin-left: 13px;
+    margin-top: 7px;
+
+    font-size: 10px;
+    font-weight: 400;
+    font-style: normal;
+    color: rgb(6, 11, 53);
+  }
+
+  &__product-price {
+    display: flex;
+
+    height: 18px;
+    font-style: normal;
+
+    margin-top: 0;
+    margin-left: 16px;
+    margin-right: 16px;
+  }
+
+  &__price-cost {
+    display: flex;
+    align-items: center;
+
+    font-size: 12px;
+    font-weight: 400;
+    color: rgb(97, 96, 112);
+  }
+
+  &__price-total {
+    display: flex;
+    align-items: center;
+    text-align: center;
+
+    width: 79px;
+    height: 18px;
+
+    margin: 0 55px;
+    margin-right: 0;
+
+    font-size: 16px;
+    font-weight: 700;
+    color: rgb(6, 11, 53);
+  }
+
+  &__buy-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+
+    width: 166px;
+    height: 40px;
+
+    margin-top: 16px;
+    margin-left: 16px;
+    margin-right: 16px;
+    margin-bottom: 17px;
```

Comment:
You should set only correct width and paddings for button, use padding for card instead of margin-left, right, bottom for this element